ci/docker/daml-sdk/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM adoptopenjdk:8-jdk-hotspot
+FROM adoptopenjdk:11-jdk-hotspot
```

Comment:
This means we also use jdk8 in docker images for any pre 2.0 release we make now, e.g., SDK 1.18.0. Should we instead do a version check in the cron job and switch depending on that?

Reply:
Should we have a 1.x set of images and a 2.x set of images for this purpose?

Reply:
We run main branch YAML files but release commit source tree, so no, as I understand the script we'd still use Java 8 for the 1.18 branch.

Reply:
sounds plausible, thanks for checking

Reply:
Specifically [here](https://github.com/digital-asset/daml/blob/main/azure-cron.yml#L84-L86):
```
              git checkout "$version"
              cd ci/docker/daml-sdk
              docker build -t digitalasset/daml-sdk:${version#v} --build-arg VERSION=${version#v} .
```

Note that this means this PR will **not** fix the current image, only the next one.

Reply:
The `adoptopenjdk` images are now deprecated in favor of [`eclipse-temurin`](https://hub.docker.com/_/eclipse-temurin/). We might want to switch to `eclipse-temurin:11-jdk`.

Reply:
Yes, thanks for raising.